StackSplit/install_StackSplit.m

```diff
@@ -142,14 +147,16 @@ function install_StackSplit()
 % check for unzipped StackSplit folder
 dirSS=dir('StackSpl*');
 
-if ~isempty(dirSS) && isfolder(dirSS.name) && length(dirSS)==1 && ~isempty(dir_orifiles)
+% if ~isempty(dirSS) && isfolder(dirSS.name) && length(dirSS)==1 && ~isempty(dir_orifiles)  % YF 2024/01/11
+if ~isempty(dirSS) && isfolder(dirSS.name) && isscalar(dirSS) && ~isempty(dir_orifiles)
```

Comment:
@michaelgrund do you want to keep the old version (length comparison) as a comment, or do you prefer to remove it? For `install_StackSplit.m` and `uninstall_StackSplit.m`, I added an explanation regarding this change to the headers of those files.

Reply:
Can be removed, that's the point of using version control and to not keep all old stuff in the code.

Reply:
Done in commit https://github.com/michaelgrund/stacksplit/pull/25/commits/08c7baab41ffcf9bbea960a0316e37c4c50ade09.

Reply:
After this change, this PR should be ready for review, if you have time.